packages/product-json-to-csv/test/map-product-data.spec.js

```diff
@@ -181,11 +185,11 @@ describe('::ProductMapping', () => {
                 name: 'color',
                 value: {
                   label: {
-                    it: 'blanco',
-                    de: 'weiss',
-                    en: 'white',
+                    it: 0,
```

Comment:
The zero value was already tested using that `lengthOfProduct` attribute so no need to test it also here in the `lenum` attribute.

Reply:
Ok, thanks